### DISCUSSION TITLE
Use Macrodroid / Tasker with VpnHood (auto-disconnect/connect)

### DISCUSSION THREAD

**Discussion by Jeje292** [vpnhood/VpnHood#543] [Ideas]
Hello,

Like OpenVPN do, is it possible to send Intent (with tool like Macrodroid or Tasker) to auto-disconnect or auto-connect VpnHood on Android ?

For example :

- Auto-Disconnect when I deactivate Wi-Fi and Mobile Data (to prevent VpnHood client to try to connect)
- Auto-Connect when I activate Wi-Fi and connect to specific SSID

I tried to search with Logcat without success

Thank you !

**Comment by trudyhood** [comment]
Thank you for suggesting these features. We need time to look at it

**Comment by trudyhood** [comment]
Moved to issue #544
